manala.ohmyzsh/templates/users/php.dev.j2

```diff
@@ -62,7 +62,7 @@ fi
 # Custom plugins may be added to ~/.oh-my-zsh/custom/plugins/
 # Example format: plugins=(rails git textmate ruby lighthouse)
 # Add wisely, as too many plugins slow down shell startup.
-{{ macros.config_row(config, 'plugins', '(git debian common-aliases history history-substring-search thefuck npm composer symfony2)') }}
+{{ macros.config_row(config, 'plugins', '(git debian common-aliases history history-substring-search thefuck npm composer symfony2, yarn)') }}
```

Comment:
space separator, not comma :)

Reply:
Hmm. My bad... Already pushed a fix?

Reply:
Nope, could you ?

Reply:
It's done in https://github.com/manala/ansible-roles/commit/6c1b2bb9b4643c910f67493f4488cfac8bb20299